yapapi/runner/__init__.py

```diff
@@ -436,16 +401,20 @@ async def fill_work_q():
                 done, pending = await asyncio.wait(
                     services.union(workers), timeout=10, return_when=asyncio.FIRST_COMPLETED
                 )
+                for task_ in done:
+                    if task_.exception():
+                        raise task_.exception()
```

Comment:
Do I understand correctly that this would stop the computation if any task failed, e.g. one of the providers disconnected?

Reply:
Hmm, you're probably right, thanks for pointing this out! It didn't realise we want to be fault-tolerant here and added this to help me debug some issues (it helped).

I'll change this just to log error, or remove this altogether, assumming the errors are already logged  -- it's out of the scope for this PR anyway.

Reply:
See also this (merged) PR:
https://github.com/golemfactory/yapapi/pull/16/files